tasks/section5.yml

```diff
@@ -91,15 +103,19 @@
       - rule_5.1.5
 
 - name: "SCORED | 5.1.5 | PATCH | Ensure permissions on /etc/cron.weekly are configured"
-  command: /bin/true
+  file:
+      dest=/etc/cron.weekly
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md